### PR TITLE
Desktop version

### PR DESCRIPTION
**Pull request by Eternalgratis**
## This is the desktop version of my portfolio project 

- [ ] Add images 
- [ ] Use linters
- [ ] Use media query

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -390,8 +399,117 @@ h4 {
...
+    grid-template-columns: 1fr 1fr 1fr;
+    gap: 24px 32px;
+    margin-top: 52px;
+  }
+
+  .about-me {
+    max-width: 100%;
+    background-image: url("./images/illustration-about-me-desktop.png");
+    background-repeat: no-repeat;
+  }
```
Comment (hiromataba) [inline style.css:463]: - [ ] Please add the `About me` text header for your about section 

Your design 👇 
Screen Shot 2022-06-09 at 23 03 44

Original Design 👇 

Screen Shot 2022-06-09 at 23 06 04

File: style.css [hunk 2]
```diff
@@ -390,8 +399,117 @@ h4 {
...
+  .example-mail {
+    display: none;
+  }
+
+  #empty-footer {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    gap: 20%;
+  }
```
Comment (hiromataba) [inline style.css]: - [ ] Please decrease the space between the contact form and the footer, 

Your design 👇 
Screen Shot 2022-06-09 at 23 08 40

The original design 👇 

Screen Shot 2022-06-09 at 23 09 54

**Review by hiromataba — CHANGES_REQUESTED** [review]
Hi @Eternalgratis    :wave: :wave: 

almost-there

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Your PR has a proper title and description  :+1: 
- Your linters are passing :heavy_check_mark:  
- Profesional README file :rocket: 

## Required Changes ♻️

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css (hunk 1 above)
Comment (Eternalgratis) [reply style.css:463]: Done

File: style.css (hunk 2 above)
Comment (Eternalgratis) [reply style.css]: Done

**Review by Kossi-stack — APPROVED** [review]
## Status: Approved ✔️
**Hi @Eternalgratis,**

You did a really great job! 💯 💯💯💯
Congratulations! 🎉 🎉 🎉 🎉 🎉


You may close this Pull Request and merge the changes
Keep being awesome dev's and good luck with your next milestone!

## Optional suggestions
- [ ] Kindly fix the background image and the paragraph
- your design
Capturetitilope
- figma design
Capturetph


_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding! 👏 👏 👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Pull request merged**